currencyii/src/main/java/nl/tudelft/trustchain/currencyii/ui/bitcoin/DAOCreateFragment.kt

```diff
@@ -103,6 +106,64 @@ class DAOCreateFragment : Fragment() {
                 AddressPrivateKeyPair("", privateKeys[0])
             )
         }
+        val vWalletFileMainNet = File(
+            this.requireContext().applicationContext.filesDir,
+            "$MAIN_NET_WALLET_NAME.wallet"
+        )
+        val vChainFileMainNet = File(
+            this.requireContext().applicationContext.filesDir,
+            "$MAIN_NET_WALLET_NAME.spvchain"
+        )
+        val vWalletFileTestNet = File(
+            this.requireContext().applicationContext.filesDir,
+            "$TEST_NET_WALLET_NAME.wallet"
+        )
+        val vChainFileTestNet = File(
+            this.requireContext().applicationContext.filesDir,
+            "$TEST_NET_WALLET_NAME.spvchain"
+        )
+
+        var fileTrailer = System.currentTimeMillis()
+        if (vWalletFileMainNet.exists()) {
+            vWalletFileMainNet.renameTo(
+                File(
+                    this.requireContext().applicationContext.filesDir,
+                    "_backup_main_net_wallet_$fileTrailer"
+                )
+            )
+            Log.w("Coin", "Renamed MainNet wallet file")
+        }
+
+        if (vChainFileMainNet.exists()) {
+            vChainFileMainNet.renameTo(
+                File(
+                    this.requireContext().applicationContext.filesDir,
+                    "_backup_main_net_spvchain_$fileTrailer"
+                )
+            )
+            Log.w("Coin", "Renamed MainNet chain file")
+        }
+
+        fileTrailer = System.currentTimeMillis()
+        if (vWalletFileTestNet.exists()) {
+            vWalletFileTestNet.renameTo(
+                File(
+                    this.requireContext().applicationContext.filesDir,
+                    "_backup_test_net_wallet_$fileTrailer"
+                )
+            )
+            Log.w("Coin", "Renamed TestNet wallet file")
+        }
+
+        if (vChainFileTestNet.exists()) {
+            vChainFileTestNet.renameTo(
+                File(
+                    this.requireContext().applicationContext.filesDir,
+                    "_backup_test_net_spvchain_$fileTrailer"
+                )
+            )
+            Log.w("Coin", "Renamed TestNet chain file")
+        }
```

Comment:
This could all be moved to a separate method, this method is getting pretty big now.